code/src/main/java/com/google/sps/storage/data-storage/NegativeUserPlace.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.storage;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import java.time.Instant;
+import java.util.ArrayList;
+
+/**
+* This class stores the information needed to identify a negative user
+* at a specific place.
+*/
+@Entity
+public class NegativeUserPlace {
+
+  private String userId;
```

Comment:
nit: here and throughout, can we order by:
Id first, indexes second, and other fields third?

Reply:
Right now I would like if userId was the index but it won't be unique

Reply:
However every other class follows that model

Reply:
addressed